src/iocore/eventsystem/Watchdog.cc

```diff
@@ -60,8 +60,9 @@ Monitor::monitor_loop() const
   // Divide by a floating point 2 to avoid truncation to zero.
   auto sleep_time = _timeout / 2.0;
   ink_release_assert(sleep_time.count() > 0);
-  Dbg(dbg_ctl_watchdog, "Starting watchdog with timeout %" PRIu64 " ms on %zu threads.  sleep_time = %" PRIu64 " us",
-      _timeout.count(), _threads.size(), std::chrono::duration_cast<std::chrono::microseconds>(sleep_time).count());
+  Dbg(dbg_ctl_watchdog, "Starting watchdog with timeout %lld ms on %zu threads.  sleep_time = %lld us",
+      std::chrono::duration_cast<std::chrono::duration<long long, std::milli>>(_timeout).count(), _threads.size(),
+      std::chrono::duration_cast<std::chrono::duration<long long, std::micro>>(sleep_time).count());
```

Comment:
Looks like the compilers are happy with it. Can we put this in a using alias though?